package.json

```diff
@@ -47,25 +47,20 @@
   ],
   "main": "./dist/extension",
   "extensionDependencies": [
+    "redhat.vscode-microprofile",
     "redhat.java",
     "vscjava.vscode-java-debug"
   ],
   "contributes": {
     "javaExtensions": [
-      "./jars/org.eclipse.lsp4mp.jdt.core.jar",
       "./jars/com.redhat.microprofile.jdt.quarkus.jar"
     ],
+    "microprofile": {
+      "jarExtensions": [
+        "./server/com.redhat.quarkus.ls-0.9.0-SNAPSHOT.jar"
```

Comment:
@fbricon if you can give me quickly how to generate this JAR without 0.9.0-SNAPSHOT it would be nice.

Reply:
fixed